app/assets/stylesheets/characters.scss

```diff
@@ -0,0 +1,19 @@
+// Place all the styles related to the characters controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
+
+.row {
```

Comment:
I don't feel like adding padding-top to every row is a good idea, may have other effects on pages you don't want